bumble/smp.py

```diff
@@ -769,19 +769,23 @@ def get_long_term_key(self, rand: bytes, ediv: int) -> Optional[bytes]:
     def decide_pairing_method(
         self, auth_req: int, initiator_io_capability: int, responder_io_capability: int
     ) -> None:
+        if self.connection.transport == BT_BR_EDR_TRANSPORT:
```

Comment:
Are you sure about this ? Can you add a comment explaining a bit + link to the spec ?

Reply:
In Core Vol 3, Part H, Chapter 3.3 Command Format, it states that only the following commands are supported on BR/EDR transport:
* Pairing Request
* Pairing Response
* Pairing Fail
* Identity Info 
* Identity Address Info
* Signing Info

Which implied that Phase 2 authentication is impossible on Classic SMP channel - just like all CTKD-related spec, it's not very clear but that's something we can infer.

Reply:
sg